index.md

```diff
@@ -0,0 +1,220 @@
+## Styleguide
+
+### Table of Contents
+
+- [1.0 General](#10-general)
+- [2.0 Angular / Coffeescript](#20-angular--coffeescript)
+  - [2.1 Basics](#21-basics)
+  - [2.2 Angular specifics](#22-angular-specifics)
+  - [2.3 Coffeescript specifics](#23-coffeescript-specifics)
+- [3.0 Django / Python](#30-django--python)
+  - [3.1 Basics](#31-basics)
+- [4.0 Styleguide](#40-styleguide)
+
+
+### 1.0 General
+
+- Avoid unnecessary else statements
+
+```python
+# bad
+if a_true_thing:
+    return True
+else:
+    return False
+
+# good
+if a_true_thing:
+    return True
+return False
+```
+
+- Avoid nested if statements
+
+```python
+# bad
+if a_true_thing:
+    if another_true_thing:
+        return 'foo'
+    else:
+        return 'bar'
+
+# good
+if a_true_thing and another_true_thing:
+    return 'foo'
+elif a_true_thing:
+    return 'bar'
+```
+
+### 2.0 Angular / Coffeescript
+
+#### 2.1 Basics
+
+- 2 space indent
+
+```coffeescript
+hello = ->
+  console.log('hi')
+  return
+hello()
+```
+
+- Single quotes for strings, unless using string interpolation, then double quotes is acceptable
+
+```coffeescript
+# bad
+aString = "Hello World"
+anotherString = '#{ aString }' # this won't work at all
+
+# good
+aString = 'Hello World'
+anotherString = "#{ aString }"
+```
+
+- Commas should have a space after them
+
+```coffeescript
+# bad
+list = [1,2,3,4]
+foo = (bar,baz) -> ...
+
+# good
+list = [1, 2, 3, 4]
+foo = (bar, baz) -> ...
+```
+
+- CamelCase when naming variables and functions
+
+```coffeescript
+# bad
+some_function = (foo, bar) -> ...
+some_var = 'Hello world'
+
+# good
+someFunction = (foo, bar) -> ...
+someVar = 'Hello world'
+```
+
+- No quotes on keys in key value pairs
+
+```coffeescript
+# bad
+obj = {'key': 'value'}
+
+# good
+obj = {key: 'value'}
+obj = {'key-with-dash': 'value'} # exception since it would be a syntax error without quotes
+
+```
+
+- Space between colon and value in key value pairs
+
+```coffeescript
+# bad
+obj = {key : 'value'}
```

Comment:
can you add a line for alignment (aka that we shouldn't do it anymore)?

Reply:
yeah, i wanted to call this out, including what to do for legacy files that havent been unaligned yet. will add that as a TODO for this pr

Reply:
VICTORY! NO ALIGNMENT!!!

Reply:
we still have a lot lingering, but no alignment in new files :D